paddlecloud/paddlejob/views.py

```diff
@@ -89,6 +91,23 @@ def post(self, request, format=None):
         # get user specified image
         job_image = obj.get("image", None)
         gpu_count = obj.get("gpu", 0)
+        # jobPackage validation: startwith /pfs
+        # NOTE: always overwrite the job package when the directory exists
+        job_package =obj.get("jobPackage", "")
```

Comment:
With this logic, users will use the parameter `jobPackage` as `/pfs/xxx/home/yyy` or `/xxxx` or a relative path, there is so much format for the jobPackage, it looks so much confusing, maybe we can only support one format, such as "/pfs/$dc/home/$username", I think it's more clearly.